src/lib/validator/GameNumber.ts

```diff
@@ -0,0 +1,71 @@
+import { ERROR_MESSAGE } from '../../constants/Messages';
+import ValidationError from '../error/ValidationError';
+
+class GameNumber {
+  #inputNumbers;
+
+  constructor(input: unknown) {
+    this.#inputNumbers = input;
+    this.#checkInput();
+  }
+
+  #checkInput() {
+    this.#checkNumber();
+    this.#checkZero();
+    this.#checkNumberOfDigits();
+    this.#checkDuplication();
+  }
+
+  #checkNumber() {
+    if (this.#isNotNumber()) {
+      throw new ValidationError(ERROR_MESSAGE.only_number);
+    }
+    this.#inputNumbers = Number(this.#inputNumbers);
+  }
+
+  #isNotNumber() {
+    return Number.isNaN(Number(this.#inputNumbers));
+  }
+
+  #checkZero() {
+    if (this.#haveZero()) {
+      throw new ValidationError(ERROR_MESSAGE.not_zero);
+    }
+  }
+
+  #haveZero() {
+    if (typeof this.#inputNumbers === 'number') {
+      return [...this.#inputNumbers.toString()].includes('0');
+    }
+    return false;
+  }
+
+  #checkNumberOfDigits() {
+    if (this.#isNotThreeDigitNumber()) {
+      throw new ValidationError(ERROR_MESSAGE.length_three);
+    }
+  }
+
+  #isNotThreeDigitNumber() {
+    if (typeof this.#inputNumbers === 'number') {
+      return this.#inputNumbers.toString().length !== 3;
```

Comment:
문제에서 3개로 고정되어있어 사실 그럴일은 없겠으나, 결국 3개인지 체크하는 이유는
숫자를 3개만 뽑기 때문이잖아요? 그렇다면 이미 선언해놓은 상수에서 불러오는게 더 적합하다고 생각해요!

이렇게 수정하면 이후에 숫자를 4개 5개 뽑더라도 따로 수정해 줄 필요가 없으니 말입니당

```suggestion
      return this.#inputNumbers.toString().length !== GENERATOR.pick_count;
```